test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MySqlCommandTests.cs

```diff
@@ -21,7 +21,8 @@ public static IEnumerable<object[]> GetMySql8Data()
         {
             foreach (object[] item in PackageVersions.MySqlData)
             {
-                if (!((string)item[0]).StartsWith("8"))
+                var version = (string)item[0];
+                if (version != string.Empty && !version.StartsWith("8"))
```

Comment:
@zacharycmontoya without this change related tests were failing locally (via `docker-compose run IntegrationTests` since the package version was always an empty string.